src/Support/ReflectionClosure.php

```diff
@@ -508,7 +508,7 @@ public function getCode()
                     break;
                 case 'id_name':
                     switch ($token[0]) {
-                        case $token[0] === ':' && $context !== 'instanceof':
+                        case $token[0] === ':' && !in_array($context, ['instanceof', 'new']):
```

Comment:
3rd param of `true` should be passed to `in_array`, to use a strict comparison.

Reply:
Done.